### PR TITLE
User can get album tracks without inputting time codes.

### PR DESCRIPTION
**Pull request by danielduffield**
Created buttons for loading the tracklist from the description, submitting a tracklist via pasting into a text box, and resetting the current tracklist.

### CODE REVIEW

**Comment by danielduffield** [comment]
youtube-issue-5

File: client/main.js [hunk 1]
```diff
@@ -81,3 +86,54 @@ const $startOverBtn = document.getElementById('start-over-button')
...
+  addTrackForm(currentTrack)
+  currentTrack++
+})
+
+const $loadTimecodesBtn = document.getElementById('load-timecodes-button')
+$loadTimecodesBtn.addEventListener('click', () => {
+  $trackFormContainer.innerHTML = ''
+  currentTrack = 1
+  addTrackForm(currentTrack)
+  console.log(albumMetadata.timeCodes, albumMetadata.timeCodes.length)
```
Comment (victormejia) [inline client/main.js]: remove this log

File: client/main.js [hunk 2]
```diff
@@ -81,3 +86,54 @@ const $startOverBtn = document.getElementById('start-over-button')
...
+  if (albumMetadata.timeCodes.length > 1) {
+    const autoTracklist = autoGenerateTracklist(albumMetadata.description, albumMetadata.videoLengthString)
+    console.log(autoTracklist)
```
Comment (victormejia) [inline client/main.js]: remove this log

File: client/main.js [hunk 3]
```diff
@@ -81,3 +86,54 @@ const $startOverBtn = document.getElementById('start-over-button')
...
+    currentTrack = autoTracklist.length + 1
+    autofillTracklistForms(autoTracklist)
+  }
+  else {
+    console.log('No timecodes in description')
```
Comment (victormejia) [inline client/main.js]: remove this log

**Pull request merged**